Composer/packages/client/src/pages/language-generation/table-view.js

```diff
@@ -231,26 +224,14 @@ export default function TableView(props) {
   }
 
   function onCopyTemplate(index) {
-    const newItems = [...templates];
-    const copyName = `${newItems[index].Name}.Copy`;
-
-    // if duplicate, increse name with Copy1 Copy2 ...
-    let repeatIndex = 0;
-    let newName = copyName;
-
-    while (templates.findIndex(item => item.Name === newName) !== -1) {
-      repeatIndex += 1;
-      newName = copyName + repeatIndex.toString();
-    }
-
+    const Name = templates[index].Name;
+    const resolvedName = lgUtil.increaseNameUtilNotExist(templates, `${Name}.Copy`);
```

Comment:
Let's use "-" or "_", instead of ".", because once we introduce ".", it has certain impact of the future work.